### PR TITLE
Update ember-cli to 2.3beta2

### PR DESCRIPTION
**Pull request by Gaurav0**
... also updating some other dependencies.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,24 +4,22 @@
     "ember": "2.3.1",
     "ember-cli-shims": "0.1.0",
     "ember-cli-test-loader": "0.2.2",
-    "ember-load-initializers": "0.1.7",
     "ember-qunit-notifications": "0.1.0",
-    "jquery": "^1.11.3",
+    "jquery": "1.11.3",
     "loader.js": "^3.5.0",
```
Comment (rwjblue) [inline bower.json]: This should be removed I think...

File: vendor/loader-shim.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* globals define */
+define('ember/load-initializers', ['exports', 'ember-load-initializers', 'ember'], function(exports, loadInitializers, Ember) {
```
Comment (rwjblue) [inline vendor/loader-shim.js]: Why do we need this shim?

File: bower.json (hunk 1 above)
Comment (Gaurav0) [inline bower.json]: Yup.

File: vendor/loader-shim.js (hunk 2 above)
Comment (Gaurav0) [inline vendor/loader-shim.js]: I suppose i included it to be thorough in including all of ember-load-initializers in the (inner) app. But I guess it is not necessary.

**Comment by Gaurav0** [comment]
@rwjblue Updated according to your comments.

**Comment by rwjblue** [comment]
Awesome, thank you!

**Pull request merged**